PubSubSpec.md

```diff
@@ -84,6 +84,14 @@ MAPPING-ENTRY-TYPE = 206
 SEQ-NO-TYPE = 204
 ```
 
+A mapping entry MAY contain additional information such as the time of publication of the data. Any additional information MUST be encoded as one or more TLV blocks following the `ApplicationName` block. SVS-PS implementations SHOULD provide a mechanism to include additional mapping information, and SHOULD allow applications to filter incoming publications based on the recevied mapping entry. Implementations MAY also provide mechanisms for automatic handling of well-known additional blocks, such as fetching only recent data using the `Timestamp`.
+
+The following well-known additional blocks are recommended.
```

Comment:
"RECOMMENDED" should be uppercase if you want to follow the RFC terminology/spelling.